Makefile

```diff
@@ -10,6 +10,8 @@
 #                shipped with libbpf-sys
 #
 ROOT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+RELEASE_VERSION ?= 1.3.0
```

Comment:
not sure this is what was intended in the feedback (some time passed), but I put this together as a potential solution and added it here. Of course, I kept it split so it's more convenient to drop it and experiment with it.
Although we now have the version here, the manifest still requires it to be set.
Feedback is very welcome.

Reply:
I think this comes from an attempt to have a version & name set based on the current git tree status and not based on the cargo target (dev vs release). Eg.
- `retis vx.y.x "name"` if the tree is pointing to a vx.y.z tag.
- `retis vx.y.z+` otherwise.

This can only be done by using git and looking at the current tree status, was experimented in `build.rs` but that turned out to be impossible to do. Now with a `Makefile` that might be possible.

Ideally we would have the version and name only defined in the `Makefile`, but I had a look at this and we can't use env variables in `Cargo.toml`. This means we would have to extract the version automatically from `Cargo.toml` (so we don't have to maintain two copies) but still set the version name in `Makefile`, which is not perfectly clean.

Reply:
you mean something like this?

`cargo metadata --format-version=1 | jq '.packages | .[] | select(.name=="retis") | .version'`

Reply:
If we agree to continue in this direction, yes, that would be nice to avoid duplicating the version in two files.

Reply:
I updated my series (and also found a way for the `release_name`) just to make it stand out and get the chance to discuss about it

Reply:
With both variables defined in `Cargo.toml` this is quite nice!